integration-tests/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -60,6 +61,9 @@
  */
 public class TfsShellTest {
   private static final int SIZE_BYTES = Constants.MB * 10;
+  @Rule
+  public LocalTachyonClusterResource mLocalTachyonClusterResource =
+      new LocalTachyonClusterResource(SIZE_BYTES, 1000, Constants.GB);
   private LocalTachyonCluster mLocalTachyonCluster = null;
```

Comment:
does this test need both 'LocalTachyonClusterResource' above, and 'LocalTachyonCluster' here?